datafusion/core/src/datasource/physical_plan/csv.rs

```diff
@@ -1191,6 +1191,8 @@ mod tests {
             Field::new("c2", DataType::UInt64, false),
         ]));
 
+        println!("{out_dir}");
```

Comment:
I think this might be left over? Since it is part of a test I think it is fine to leave in, just thought I would point it out